src/couchapps/WMStats/_attachments/js/minified/global.min.js

```diff
@@ -66,6 +67,16 @@ WMStats.Globals = function($){
     
     function getReqDetailPrefix () {
         if (_dbVariants[dbname] == "tier1") {
```

Comment:
I am not sure analysis one is used (may be not)  but we have to distinguish between tier0 and production.